### PR TITLE
Added Swedish Translation

### PR DESCRIPTION
**Pull request by bittin**

### CODE REVIEW

**Review by beanpole135 — CHANGES_REQUESTED** [review]
All of these translations are in the wrong spot and will not be used they way they are now.
The `type="unfinished"` tag needs to be removed, and the translated text put between the tags like this:
`<translation>This is translated text</translation>`

**Comment by bittin** [comment]
fixed

File: src-qt5/desktop-utils/lumina-mediaplayer/i18n/l-mediap_sv.ts [hunk 1]
```diff
@@ -7,427 +7,427 @@
...
         <location filename="../mainUI.ui" line="479"/>
         <source>Delete current station</source>
-        <translation type="unfinished"></translation>
+        <translation>Ta bort nuvarande station</translation>
     </message>
     <message>
         <location filename="../mainUI.ui" line="482"/>
         <source>rm</source>
-        <translation type="unfinished"></translation>
+        <translation>=rm</translation>
```
Comment (beanpole135) [inline src-qt5/desktop-utils/lumina-mediaplayer/i18n/l-mediap_sv.ts:92]:
```suggestion
        <translation>rm</translation>
```

**Review by beanpole135 — APPROVED** [review]
Looks good!

**Pull request merged**